plugin-manifest.ts

```
@@ -1,5 +1,7 @@
-import { EncodedExtension } from '@openshift/dynamic-plugin-sdk-webpack';
-import { ConsolePluginBuildMetadata } from '@openshift-console/dynamic-plugin-sdk-webpack/lib/build-types';
+import {
+  EncodedExtension,
+  ConsolePluginBuildMetadata,
```

ESLint

[error] Expected "ConsolePluginBuildMetadata" to come before "EncodedExtension"

(perfectionist/sort-named-imports)